config/kibana.yml

```diff
@@ -110,3 +110,4 @@
 # Specifies locale to be used for all localizable strings, dates and number formats.
 # Supported languages are the following: English - en , by default , Chinese - zh-CN .
 #i18n.locale: "en"
+
```

Comment:
unnecessary change?

Reply:
Yes , sorry about it being in PR, it's still very much work in progress

Reply:
Github's UI currently does an incredible bad job at informing the user when they automatically request codeowners for review by simply opening a PR:

![image](https://user-images.githubusercontent.com/209966/76312582-a5f2dd80-62d3-11ea-8715-61757bdbad88.png)

I've written to Github Support to get them to add some kind of warning like: "Creating this PR will ping 140 people - do you wish to continue?". Until they add that we'll have to stick to Draft PRs (even though it's easy to forget).

If you agree with the above warning suggestion feel free to pile on and write to Github Support about this so they prioritize it :)

Reply:
Yeah this should've been kept in draft before finalizing changes, will keep in mind about those notifications. Thank you for patience, and i am also unable to remove you guys for reviewers.

Reply:
i will close this PR and create a new draft.